README.md

```diff
@@ -14,6 +14,23 @@ By default there is little/no output to stdout or stderr until the script has fi
 
 When using `jellyfin.py`, the results can be saved to `json` using the `-j` parameter. These will be saved in a sub-directory in `pwd`. Saving the results as json also allows them to be checked in subsequent runs to skip already processed files.
 
+## Running Docker
+  ### Scanner
+  ```
+  docker run -d \
+  --name=Jellyfin-intro-Scanner \
+  -e JELLYFIN_URL=http://Jellyfin:port \
+  -e JELLYFIN_USERNAME=username \
+  -e JELLYFIN_PASSWORD='password' \
```

Comment:
I've added `CONFIG_DIR` and `DATA_DIR` as supported env variables.

By default, config dir is at `./config` and stores `path_map.txt` and the data dir
By default, data dir is `os.path.join(config_dir, 'data')` and it stores `fingerprints` and `jellyfin_cache`